src/averageAges.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+/**
+ * Helper Functions
+ */
+const filterByGender = (people, gender) => {
+  return people.filter((person) => person.sex === gender);
+};
+
+const calculateAverage = (array) => {
+  return array.reduce((a, b) => b.died - b.born + a, 0) / array.length;
```

Comment:
one-letter variables are not great
```suggestion
  return array.reduce((ageSum, {died, born}) => (
    died - born + ageSum
  ), 0) / array.length;
```